xml/System.Security.Cryptography/FromBase64Transform.xml

```diff
@@ -49,7 +49,10 @@
    
   
 ## Examples  
- The following example decodes a base 64-encoded file to an output text file.  
+ The following example decodes a base 64-encoded file to an output text file. 
+
+> [!NOTE]
+>  On .NET implementations where `FromBase64Transform.InputBlockSize` is not `4`, a constant value of `4` should be substituted for the property value.  
```

Comment:
Is there any way to specify that before .NET 5 the inputblock size = 1?
So it is more obvious to users.

Reply:
We can open an issue to track that change for .NET 5 but we wouldn't add this yet